_pages/about.md

```diff
@@ -324,6 +324,18 @@ Recognition and Sentiment Classification". [&nbsp;<a href = "http://ir.hit.edu.c
     <td>
       <p>
         <strong>Harms</strong><br/>
+        <iframe width="560" height="315" src="https://www.youtube.com/embed/JYY45-0te_U?si=oqgT-22M91ui0CGa" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
+      </p>
+      <p>
+        Lecturers: Minzhi Li, Xinyuan Lu, Taha, Richmond Sin, Shivshankar Umashankar, Yuxi Xie, Cecilia Soh, Li keyou, Tan Yi Ting<br/>
+      </p>
+      <p>
+        Supporters: Ng Hui Ling, Tiviatis, Reuben Thomas, Guanzhen Li, Hannah Brown<br/>
```

Comment:
Tiviatis Sim

Please fix the above typo in the line.  Thanks!

Reply:
Sorry for my typo. I've already pushed the fixed lines to the repo. Hope you can see the update in this PR?